integrations/openai/src/unitycatalog_ai_openai/toolkit.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any, Dict, List, Optional
+
+from openai.lib._tools import pydantic_function_tool
```

Comment:
Do we need to use this? This makes the openai integration fragile.

Reply:
This is actually exposed at openai level, let me change the import :)